yarn.lock

```diff
@@ -2377,11 +2382,6 @@ wcwidth@^1.0.1:
   dependencies:
     defaults "^1.0.3"
 
-web-tree-sitter@0.19.1:
```

Comment:
Curious about this

Reply:
this is a good thing—tbh I'm surprised this was here cc/ @wenkokke